app/Services/WebResourceRetrieverService.php

```diff
@@ -79,9 +79,15 @@ public function getStudent($email)
         $client = new Client();
 
         return $client->get(
-            env('DIRECTORY_URL') . $email . '?secret=' . urlencode(env('DIRECTORY_SECRET_KEY')),
+            env('PROD_DIRECTORY_URL') . 'members?email=' . $email,
```

Comment:
This change won't allow you to retrieve student's info, a simpler thing to do here is to make an if check to see if the app is in production mode or not and update the URL passed into the `get()`